app/models/user.rb

```diff
@@ -517,6 +517,16 @@ def self.recently_active_users(limit = 5, order = 'last_updated DESC')
     end
   end
 
+  def drafts
+    Node.where(uid: uid)
+    .where(status: 3, type: 'note')
+    .order('created DESC')
+  end
+
+  def drafts_count
+    drafts.size
```

Comment:
Couldn't the view code just run `user.drafts.size`?

Reply:
You are right!